main/src/ca/uwaterloo/flix/api/lsp/provider/SemanticTokensProvider.scala

```diff
@@ -625,6 +625,21 @@ object SemanticTokensProvider {
   private def visitExps(exps0: List[Expr]): Iterator[SemanticToken] =
     exps0.flatMap(visitExp).iterator
 
+  private def visitCst(constant: Expr.Cst): Iterator[SemanticToken] = constant.cst match {
```

Comment:
I think we should just inline the function, since it is only used in one place. Is that right?